Fix/issue 2712 - Make The Tariff Number Field a Required Field when the Destination is Within EU (#2764)
Automattic/woocommerce-services: changelog.txt

*** WooCommerce Shipping & Tax Changelog ***

= 2.6.2 - 2024-xx-xx =

> * Fix - Tariff number field on customs should be required field when the destination country is within EU.

Reviewer: ```suggestion
* Fix - Require HS Tariff number on customs form for EU destination countries.
```
